test/functionalTest/cases/1308_subscription_cache/sub_expired_and_updated_then_cache_refresh.test

```diff
@@ -289,9 +289,9 @@ Date: REGEX(.*)
 
 {
     "ids": "REGEX(.*)",
-    "inserts": REGEX((2|3)),
+    "inserts": REGEX((2|3|4)),
```

Comment:
Nothing to do with the PR, correct.
This functest failed sporadically.
Now it should not fail anymore ...

NTC